spec/tree_spec.rb

```diff
@@ -5,10 +5,25 @@
   it 'should be a Class' do
     expect(described_class.is_a? Class).to eq true
   end
+  it 'should create an instance of Tree with correct attributes' do
+    tree = Tree.new(5, 1, 0, true)
+    expect(tree.height).to eq(5);
+    expect(tree.age).to eq(1);
+    expect(tree.apples).to eq(0);
+    expect(tree.alive).to eq(true);
+  end
 end
 
-describe 'Fruit' do
+describe Fruit do
+  it 'should be a Class' do
+    expect(described_class.is_a? Class).to eq true
+  end
```

Comment:
This kind of test is quite possibly not worth having in a project.  If `Fruit` isn't a class, then all kinds of stuff should be blowing up in your tests.

Reply:
I've changed it to be two tests, one for Apple < Fruit, and one for apple to be_an_instance_of(Apple)